CMakeLists.txt

```diff
@@ -1,13 +1,22 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 3.1)
 
-project(BATCHEDMAT LANGUAGES CXX)
+project(NEML2 LANGUAGES CXX)
 
 # ## Setup modules ###
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 
 # Werror, etc
 add_compile_options(-Wall -Wextra -pedantic -Werror)
 
+# This is aggravating but occurs because pre-packaged torch uses
```

Comment:
I guess this is fine for now, leave it as off because right now we're getting the test version from pip.  But we'll change this once someone changes to getting libtorch via a script for testing.